src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -932,6 +963,7 @@ describe('CdkVirtualScrollViewport', () => {
 
       viewport.setTotalContentSize(10000);
       flush();
+      fixture.changeDetectorRef.markForCheck();
```

Comment:
This looks suspicious to me. Shouldn't `setTotalContentSize` call `markForCheck`  in the `CdkVirtualScrollViewport`?  Same question for `setRenderedRange`